src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
```

Comment:
When creating a new instance of a figure, it's better to avoid passing expressions directly into the constructor for better readability. Consider assigning 'radius' and 'color' to variables before passing them to the constructor.